src/components/virtualRepeat/demoDeferredLoading/index.html

```diff
@@ -0,0 +1,22 @@
+<div ng-controller="AppCtrl as ctrl">
+  <md-content layout="column">
+    <p>
+       Display a list of 50,000 items that load on demand in a viewport of only 7 rows (height=40px).
+       <br/><br/>
+       This demo shows scroll and rendering performance gains when using <code>md-virtual-repeat</code>;
+       achieved with the dynamic reuse of rows visible in the viewport area. Developers are required to
+       explicitly use <code>md-virtual-repeat-container</code> as a wrapping parent container.
+       <br/><br/>
+       To enable load-on-demand behavior, developers must pass in a custom instance of
+       mdVirtualRepeatModel (see the example's source for more info).
```

Comment:
Also, md-infinite isn't quite right... infinite scroll is one of two behaviors unlocked by this feature (hence the two demos).

md-deferred? md-deferred-data? md-deferred-load?